frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -319,28 +339,19 @@ const TestCardForm = ({
     if (testResultsError) {
       showError(testResultsError, "Invalid test results");
     }
-    if (aoeError) {
-      showError(aoeError, "Invalid test questionnaire");
-    }
     return (
       dateTestedError === null &&
       deviceTypeError === null &&
       specimenTypeError === null &&
-      testResultsError === null &&
-      aoeError === null
+      testResultsError === null
```

Comment:
pulling out the aoe validation away from the rest of the test card here since there was enough complication in just that piece to warrant its own set of error handling